### DISCUSSION TITLE
OpenKruise Helm Chart部署咨询

### DISCUSSION THREAD

**Discussion by wangtianlong1021** [openkruise/kruise#1458] [Q&A]
环境：腾讯TKE 1.20 docker  centos
在values文件中：socketLocation: "/var/run"  能否只挂载sock文件，在我们的环境中遇到一个问题，
netns正好在/var/run下，在我们删除Pod时，会报"error:
Unknown desc = failed to get sandbox ip: check network namespace closed: remove netns: unlinkat /var/run/netns/cni
rpc error:
code
770ae501-20a2-6a80-65a9-583b7ee231a1: device or resource busy"  因为我们近期没有进行集群过大的变更，OpenKruise是在上周部署测试的，而Kruise-daemonset 正好挂载了/var/run
相同的情况：https://www.boysec.cn/boy/f0530e00.html

**Comment by furykerry** [comment]
if we mount only the socket file, if the socket file changes the mount will not load the changed file

**Reply by wangtianlong1021** [reply]
但是在docker运行时，socketLocation: "/var/run/docker.sock"，kruise-daemonset会报错 Failed to new daemon: failed to new runtime factory: not found container runtime sock
